src/components/app.css

```diff
@@ -8,46 +8,144 @@
 }
 .northy {
     background-image: url('../assets/north.png');
-  background-size: cover;
-  font-size: 1.45em;
-  font-weight: bold;
-  display: flex;
-  text-align: center;
-  box-shadow: 2px 2px 16px 1px rgba(0, 0, 0, 0.3);
-    padding: 60px;
+   border-left: solid darkblue 2px;
 }
 
 .southy { 
-    background-image: url('../assets/south.png');
-  background-size: cover;
-  font-size: 1.3em;
-  font-weight: bold;
+  background-image: url('../assets/south.png');
+}
+
+.westy {
+  background-image: url('../assets/west.png');
+}
+
+.easty {
+background-image: url('../assets/east.png');
+}
```

Comment:
- [ ] Kindly be noticed that you should add `background-repeat: no-repeat;` and `background-size: cover;` so that it looks professional.

Before | After
--------- | -------
![Screenshot from 2022-08-02 01-37-15](https://user-images.githubusercontent.com/33752735/182262693-e9840fa9-1a64-4e92-bb88-2fe2d5397d77.png) | ![Screenshot from 2022-08-02 01-36-39](https://user-images.githubusercontent.com/33752735/182262706-a81c24ff-b0d4-4e62-ae95-1c659a1e274c.png)